roles/libvirt_manager/tasks/start_vms.yml

```diff
@@ -49,12 +49,8 @@
       {{
         _cifmw_libvirt_manager_layout.vms[vm_type]
       }}
-    extracted_ip: >-
-      {{
-        _libvirt_manager_networking.instances[vm].networks[cifmw_libvirt_manager_pub_net]['ip_v4']
-      }}
   ansible.builtin.wait_for:
-    host: "{{ extracted_ip }}"
+    host: "{{ vm }}.utility"
```

Comment:
molecule needs an update to switch DNS to dnsmasq + delegate `utility` to our instance. Will debug that locally once my current deploy is over.